create-matrix.js

```diff
@@ -76,11 +76,20 @@ directories.forEach(library => {
     const dependencies = dependant.get(library);
     if (dependencies) {
         let info = library;
-        const deps = [];
+        let deps = [];
+        // TODO check the conflict
         dependencies.forEach((dependency) => {
-            if (dependant.get(dependency))
-                deps.push(dependency);
+            if (!deps.includes(dependency)) {
+                deps.unshift(dependency);
+            }
         });
+
+        // HACK: temporary fix openssl & openssl-1.1.1 confliction
```

Comment:
so i guess it's okay when **you're** doing temporary hacks?

Reply:
no i don't like it. imo current time, it only problem of python3. but case can be increased.

so, we have to resolve vdpm part & update VITASDK package. but i can't handle it that because my free time issue.
can you contribute it?

Reply:
No

Reply:
so, we have to accept this hacky solution.